css/styled.css

```
@@ -435,8 +438,198 @@ body {
   padding-top: 25px;
   padding-left: 18px;
 }
+/* END information section */
 
 
+/* START featured section */
+
+.featured {
+  padding-top: 50px;
+  padding-left: 100px;
+  padding-right: 100px;
+}
+
+.featured__title {
+  color: #363535;
+  font-size: 24px;
+  font-weight: 700;
+  margin-bottom: 30px;
+}
+
+.featured__container {
+  display: flex;
+  gap: 20px;
+}
+
+.featured__products-info {
```

Yo a esta le pondria `featured__products`
```suggestion
.featured__products{
```